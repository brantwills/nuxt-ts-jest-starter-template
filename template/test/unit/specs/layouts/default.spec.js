import { shallow } from '@vue/test-utils'
import { createRenderer } from 'vue-server-renderer'
import Layout from '~/layouts/default.vue'

describe(`Layout default`, () => {

  it('renders to a snapshot', () => {
    const renderer = createRenderer()
    const wrapper = shallow(Layout, {})
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err)
      expect(str).toMatchSnapshot()
    })
  })

})