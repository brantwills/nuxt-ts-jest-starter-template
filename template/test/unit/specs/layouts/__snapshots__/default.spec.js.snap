// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout default renders to a snapshot 1`] = `
<div>
    <nuxt></nuxt>
</div>
`;
